test/test_debugger.cpp

```diff
@@ -0,0 +1,167 @@
+/***************************************************************************
+* Copyright (c) 2018, Martin Renou, Johan Mabille, Sylvain Corlay and      *
+* Wolf Vollprecht                                                          *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#include <chrono>
+#include <cstdlib>
+#include <fstream>
+#include <thread>
+
+#include "gtest/gtest.h"
+#include "xeus_client.hpp"
+
+namespace nl = nlohmann;
+using namespace std::chrono_literals;
+
+/***********************
+ * Predefined messages *
+ ***********************/
+
+nl::json make_attach_request()
+{
+    nl::json req = {
+        {"type", "request"},
+        {"seq", 1},
+        {"command", "attach"}
+    };
+    return req;
+}
+
+nl::json make_init_request()
+{
+    nl::json req = {
+        {"type", "request"},
+        {"seq", 1},
+        {"command", "initialize"},
+        {"arguments", {
+            {"cliendID", "vscode"},
```

Comment:
This looks like a typo

Reply:
Well this is just a cliendID example in the tests, I don't think it really matters.